ethereum/executionlayer/src/test/java/tech/pegasys/teku/ethereum/executionlayer/CapellaExecutionClientHandlerTest.java

```diff
@@ -110,6 +111,26 @@ void engineForkChoiceUpdated_shouldCallEngineForkChoiceUpdatedV2() {
     verify(executionEngineClient).forkChoiceUpdatedV2(forkChoiceStateV1, payloadAttributes);
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
+  @Test
+  void engineForkChoiceUpdated_shouldCallEngineForkChoiceUpdatedV1() {
```

Comment:
The name kinda sounds like we expect to always call FCU_v1. Might be worth adding a bit of flavour on when we expect this to happen.

Reply:
fair, updated.